recipes/qfit/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "qfit" %}
+{% set version = "2.0.5" %}
+{% set sha256 = "dfdd267742850d7e3a0ddf1ba1cd1ca202307360a3fa7168c3413dedf56834b0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/qfit-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```

Comment:
Please have a look at:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

You'll want to set a jinja variable `{% set python_min = "3.10" %}` and then use `python_min` in `host`, `run` and `test` as described in the docs.